tests/render/test_default_markdown_view.py

```diff
@@ -0,0 +1,823 @@
+import json
+import os
+from collections import OrderedDict
+
+import pytest
+
+import great_expectations as ge
+import great_expectations.render as render
+from great_expectations.core import ExpectationSuiteValidationResultSchema
+from great_expectations.render.page_renderer_util import (
+    render_multiple_validation_result_pages_markdown,
+)
+from great_expectations.render.renderer import ValidationResultsPageRenderer
+from great_expectations.render.types import (
+    RenderedDocumentContent,
+    RenderedHeaderContent,
+    RenderedSectionContent,
+    RenderedTableContent,
+)
+from great_expectations.render.view import DefaultMarkdownPageView
+from great_expectations.validation_operators.types.validation_operator_result import (
+    ValidationOperatorResult,
+)
+
+
+@pytest.fixture()
+def validation_results():
```

Comment:
I'd rename fixture to `validation_operator_result` to be more clear, since validation operator result can have many "validation results".